### ISSUE TITLE
Hasura Cloud Metrics – "Please click here to enable access to the monitoring data"

### ISSUE THREAD

**Issue by lapidus** [hasura/graphql-engine#7014]
Whenever navigating to the /console/pro/ we have to re-enable access to the page. This happens a few times per day:
`Please click here to enable access to the monitoring data.`

![image](https://user-images.githubusercontent.com/66601/120916952-e1ce4b80-c6ac-11eb-9e52-12ba4d5da4ed.png)

This also hinders accessing direct links like the below:
https://cloud.hasura.io/project/XXXXXXXXXX/console/pro/error?filters=%5B%7B%22value%22:%7B%22start%22:%222021-06-02T22:00:00.000Z%22,%22end%22:%222021-06-04T22:00:00.000Z%22%7D,%22type%22:%22time_range%22%7D%5D&groupbys=%5B%22error_code%22,%22operation_name%22,%22operation_type%22,%22operation_id%22,%22transport%22,%22user_role%22,%22client_name%22%5D

Basically any link will take you to the prompt to re-enable access.

**Comment by soorajshankar** [comment]
Hello @lapidus,

Thanks for submitting the issue! 
we are working on improvising this UX and make it seamless, we will update here when we release this feature.
